src/app/pages/project/results/results-grid/results.grid.component.ts

```diff
@@ -206,7 +210,7 @@ export class ResultGridComponent implements OnInit {
     const queryParam = {};
     queryParam['selectedResult'] = testResultId;
     this.router.navigate([], { queryParams: queryParam, queryParamsHandling: 'merge' }).then(() => {
-      this.router.navigate(['/project/' + this.route.snapshot.params['projectId'] + '/testresult/' + testResultId]);
+      this.router.navigate(['/project/' + this.projectId + '/testresult/' + testResultId]);
```

Comment:
would you like use it like 
`/project/${this.projectId}/testresult/${testResultId}`